### ISSUE TITLE
mzid filter to adjust FDR

### ISSUE THREAD

**Issue by Sjan1** [lgatto/rtslprot#1]
This filter needs a certain level of flexibility for optimisation and visualizaiton - I like the FDR, miscleavage, errors of precursor mass distribution plots for the individual mzid files.
To have this flexibility, we would better leave the filtering and optimisation in R script, rather than have it here in rtslprot package.
If we leave the peptide filter there, we will have to save the adjusted mzids, optimise and plot it, then in another loop, proceed with their combination. Correct?

**Comment by lgatto** [comment]
I agree that flexibility is important, but if you set a FDR threshold and systematically stick with it, letting the software decide the thresholds based, you will be able to compare results across different experiments. You could always go back and adapt if necessary. All the QC plots can always be generated, as this is done before applying the filters.

**Comment by Sjan1** [comment]
I was looking at the examples, and if I think I understand what we need: to create the filtObj (i) then use it in optimize_filter' (ii), then apply_filter (iii). How to add it to the rtslprot package? There are quite a few parameters involved. Would it be simpler to run the filter (i, ii, iii) on all the mzid files separately before starting 'make_pep_MSnSet'? In other words moving it outside rtslprot?
